### PR TITLE
Register WebHost configuration with ${configsetting} and LoggingProvider-options

### PR DESCRIPTION
**Pull request by snakefoot**
Updated to AspNetCore ver. 2.1 that has Long-Term-Support (LTS)

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ assembly_info:
   patch: true
   file: '**\AssemblyInfo.cs'
   assembly_version: '4.0.0'
-  assembly_file_version: '4.7.1.{build}' #NLog.Web
-  assembly_informational_version: '4.7.1' #NLog.Web
+  assembly_file_version: '4.8.0.{build}' #NLog.Web
```
Comment (304NotModified) [inline appveyor.yml]: 4.8 because the update of the dependency?  So skip 4.7.2?

File: NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj [hunk 2]
```diff
@@ -30,10 +30,10 @@ Supported platforms:
...
     <PackageTags>logging;log;session;NLog;web;aspnet;aspnetcore;MVC;httpcontext</PackageTags>
     <PackageReleaseNotes>
-- Added ThreadSafe-attribute for LayoutRenderer to optimize async Precalculate (@snakefoot)
-- ${AspNetRequestIp} added CheckForwardedForHeader to check for X-Forwarded-For header (#325) (@Giorgi)
-- Check that Content-Type is correct before accessing the Form (#322) (@mcliment)
-- Enabled SymbolPackageFormat=snupkg (@snakefoot)
+- Updated Microsoft.AspNetCore to ver. 2.1 (Long-Term-Support)
+- ${configsetting} layout renderer now depends on UseNLog-call to provide IConfiguration using IServiceProvider
+- NLogLoggerProvider options now binds to IConfiguration (Using section Logging:NLog)
+- UseNLog-call now also available for IHostBuilder
```
Comment (304NotModified) [inline NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj]: this one isn't released yet: https://github.com/NLog/NLog.Web/milestone/29?closed=1

File: appveyor.yml (hunk 1 above)
Comment (snakefoot) [reply appveyor.yml]: Should I keep NLog.Web at 4.7.1 ?

Comment (snakefoot) [reply appveyor.yml]: Reverted version change for NLog.Web

**Comment by 304NotModified** [comment]
Thanks!

**Pull request merged**